code/controllers/subsystems/ticker.dm

```diff
@@ -0,0 +1,537 @@
+SUBSYSTEM_DEF(ticker)
```

Comment:
where did you get this from? How is it better than the multifile goon system we had before?

Reply:
> where did you get this from? How is it better than the multifile goon system we had before?

Base tickers tgstation/TGMC/Baystation